message-analyzer/PostWithSentiment.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace MessageAnalyzer
+{
+    internal class PostWithSentiment : Post
+    {
+        public PostWithSentiment() { }
+
+        public PostWithSentiment(Post p)
+        {
+            this.CorrelationId = p.CorrelationId;
+            this.MessageId = p.MessageId;
+            this.Message = p.Message;
+            this.CreationDate = p.CreationDate;
+        }
+
+        // like "stars": 1-5
+        public int Sentiment { get; set; }
```

Comment:
I thought this would be string because sentiment is used as key name in actor state management

Reply:
I'm ok with making it a number as a string "1", "2", etc.  or were you expecting something else?